gwsumm/data/utils.py

```diff
@@ -26,7 +26,7 @@
 except ImportError:
     from ordereddict import OrderedDict
 
-from glue.segments import segmentlist as GlueSegmentList
+from ligo.segments import segmentlist as GlueSegmentList
```

Comment:
@alurban, just because I'm pedantic, can you rename the imported object:

```suggestion
from ligo.segments import segmentlist as LigoSegmentList
```

Reply:
@duncanmmacleod, I don't know how I didn't catch that before but the name would have bothered me too. Thanks for picking up on it!